### PR TITLE
add support for CAPA

### PR DESCRIPTION
**Pull request by bdehri**
This PR introduces a new psp for allowing projected volumes. Towards https://github.com/giantswarm/giantswarm/issues/26467

### CODE REVIEW

File: helm/aws-load-balancer-controller/templates/psp.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+apiVersion: policy/v1beta1
```
Comment (fiunchinho) [inline helm/aws-load-balancer-controller/templates/psp.yaml:1]: Should this be wrapped around a conditional value to help during the migration to PSS or is that not necessary here?

Comment (bdehri) [reply helm/aws-load-balancer-controller/templates/psp.yaml:1]: If we are going to use the same approach as vintage, I can add the global value here as well.

Comment (fiunchinho) [reply helm/aws-load-balancer-controller/templates/psp.yaml:1]: Oh I thought this app was used on `vintage`. I don't think we have decided on an approach for CAPI /cc @T-Kukawka

File: helm/aws-load-balancer-controller/values.yaml [hunk 2]
```diff
@@ -281,3 +281,7 @@ clusterSecretsPermissions:
 
 verticalPodAutoscaler:
   enabled: true
+
+global:
+  podSecurityStandarts:
```
Comment (T-Kukawka) [inline helm/aws-load-balancer-controller/values.yaml:286]:
```suggestion
  podSecurityStandards:
```

File: helm/aws-load-balancer-controller/templates/psp.yaml [hunk 3]
```diff
@@ -1,3 +1,4 @@
+{{- if eq .Values.global.podSecurityStandarts.enforced "false" }}
```
Comment (T-Kukawka) [inline helm/aws-load-balancer-controller/templates/psp.yaml:1]:
```suggestion
{{- if eq .Values.global.podSecurityStandards.enforced "false" }}
```

**Review by calvix — APPROVED** [review]

**Review by AndiDog — APPROVED** [review]

**Review by fiunchinho — APPROVED** [review]

**Pull request merged**